packages/connect-common/files/coins.json

```diff
@@ -1404,51 +1384,6 @@
             "xpub_magic_segwit_native": null,
             "xpub_magic_segwit_p2sh": null
         },
-        {
-            "address_type": 38,
-            "address_type_p2sh": 53,
-            "bech32_prefix": "nix",
-            "blockchain_link": null,
-            "blocktime_seconds": 120,
```

Comment:
Same as before, this was deleted. I am not sure if this is the correct behavior of the script `scripts/update-coins.sh`.